CIP-XXXX/README.md

```diff
@@ -0,0 +1,377 @@
+**MUST** be defined as part of the budget proposal.
+
+##### Rationale
+
+It is imperative that the withdrawn Treasury funds be protected from exorbitant
+or malicious administrative costs that severely deplete the spendable funds. In
+alignment with the goal of community transparency, it's important that all
+administrative costs of budget management be disclosed as part of the budget
+process.
+
+<hr />
+
+#### Rule: Applicable Period
+
+##### Definition
+
+A budget proposal **MUST** specify its validity period. A budget proposal
+**MUST NOT** have a validity period longer than _net change period_ time.
+
+Treasury withdrawals referencing a specific budget **MUST** occur within the
+budget's specified validity period.
+
+##### Rationale
+
+Given that governance actors (DReps and CC) are tasked with ensuring that
+treasury withdrawals align with a specified budget and do not change the
+Treasury balance by greater than _net change limit_ within _net change period_
+and that there may be one or more budgets active at any given time, it is
+imperative that a given budget proposal specify the time period during which it
+should be considered valid.
+
+This provides an easy first point of indexing for governance actors to verify
+withdrawal validity and also prevents treasury withdrawals against historical
+budgets outside their validity period.
+
+<hr />
+
+#### Rule: Maximum Spend and Surplus Definitions
+
+##### Definition
+
+A budget proposal **MUST** define an absolute maximum spend amount (denominated
+in Ada). If a budget is delineated into one or more spending categories
+(buckets) then the budget **MUST** specify a maximum spend per category in
+addition to a maximum spend for the total budget. A budget proposal **MUST**
+define what will be done with any surplus funds at the end of the budget
+validity period. A budget proposal **SHOULD** propose to return surplus funds to
+the Treasury.
+
+##### Rationale
+
+Given that budgets are generally estimated ranges of spending, it is possible
+that multiple withdrawals may be made against the same budget throughout its
+validity period. To increase transparency, a budget must define the maximum
+amount that may be spent against the entire budget and any specifically
+earmarked categories or buckets.
+
+Given that withdrawals will likely be made in large quarterly or semi-annual
+batches, it is critical that a budget proposal address the plan of action should
+there be any surplus funds remaining at the end of the budget validity period.
+It is recommended that budget proposers should return any surplus funds to the
+Treasury so these funds may recirculate into future budgets, but this is not
+mandatory to leave room for creative surplus management strategies to be
+developed and/or proposed.
+
+<hr />
+
+#### Rule: Withdrawals Reference Budgets
+
+##### Definition
+
+Every _Treasury Withdrawal Action_ **MUST** reference a ratified and currently
```

Comment:
A Treasury Withdrawal Action can reference MULTIPLE ratified & currently valid Budget Info Action since we combine those withdrawal to reduce number of votings for DRep & CC.

Reply:
Maybe...
```
Every _Treasury Withdrawal Action_ **MUST** reference one or more ratified and currently...
```
??? We'll need to add a bit more language here as well to specify that the withdrawal should specify which outputs map to which budget(s)?